packages/chat/src/components/SystemResponse/SystemMessage.tsx

```diff
@@ -83,7 +84,14 @@ export const SystemMessage: React.FC<SystemMessageProps> = ({
             <div className={messageContainer}>
               {match(message)
                 .with({ type: MessageType.TEXT }, ({ text, ai }) => (
-                  <AgentMessage text={text} ai={ai} isLast={isLast} feedback={feedback} textContent={textContent} />
+                  <AgentMessage
+                    text={text}
+                    generatedMessage={aiMessage}
```

Comment:
I would call this prop `disclaimer` or `disclaimerMessage` - I really didn't understand what this was at first..